lib/Capabilities.php

```diff
@@ -247,10 +247,6 @@ public function getCapabilities(): array {
 			$capabilities['config']['signaling']['hello-v2-token-key'] = $pubKey;
 		}
 
-		if ($this->serverConfig->getAppValue('spreed', 'has_reference_id', 'no') === 'yes') {
-			$capabilities['features'][] = 'chat-reference-id';
```

Comment:
The capability needs to be always on, so add it to the hardcoded feature list 🤔

Reply:
Now after thinking about it, maybe we should adjust the docs in https://github.com/nextcloud/spreed/blob/fb8a73c011802429138d5702ec1041fdfb81f780/docs/capabilities.md#L52 to point out that it's always set since version X